### PR TITLE
Cleanup JanusGraph-dist module

### PR DESCRIPTION
**Pull request by farodin91**
* Remove dockerfile and build process
* Remove pkg files and build process

Signed-off-by: Jan Jansen <jan.jansen@gdata.de>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### CODE REVIEW

File: janusgraph-dist/src/assembly/descriptor/common-full.component.xml [hunk 1]
```diff
@@ -64,15 +64,5 @@
                 <exclude>${project.groupId}:janusgraph-core:jar:javadoc:${project.version}</exclude>
             </excludes>
         </dependencySet>
-
-        <!-- Example data (from janusgraph-hadoop/faunus) -->
-        <dependencySet>
```
Comment (FlorianHockmann) [inline janusgraph-dist/src/assembly/descriptor/common-full.component.xml:69]: Could you briefly explain what the purpose of this was? Was it part of the packaging?

**Review by FlorianHockmann — APPROVED** [review]
LGTM, I just added one question of understanding.

File: janusgraph-dist/src/assembly/descriptor/common-full.component.xml (hunk 1 above)
Comment (farodin91) [reply janusgraph-dist/src/assembly/descriptor/common-full.component.xml:69]: I have no idea.

Comment (FlorianHockmann) [reply janusgraph-dist/src/assembly/descriptor/common-full.component.xml:69]: But you are sure that we don't need it any more?

Comment (farodin91) [reply janusgraph-dist/src/assembly/descriptor/common-full.component.xml:69]: The folder doesn't exists.

Comment (farodin91) [reply janusgraph-dist/src/assembly/descriptor/common-full.component.xml:69]: I will check something.

Comment (farodin91) [reply janusgraph-dist/src/assembly/descriptor/common-full.component.xml:69]: Looks good.

**Pull request merged**